extensions/jira/src/helpers/issues.ts

```diff
@@ -315,3 +315,17 @@ export function getCustomFieldValue(fieldSchema: CustomFieldSchema, value: unkno
       return null;
   }
 }
+
+export function generateBranchName(issue: Issue | IssueDetail, nameFormat: string): string {
```

Comment:
Should we have a check if the user removes the value from the preference that it still shows the default?